pkg/platform/packet/packet_test.go

```diff
@@ -55,6 +55,25 @@ func TestCheckWorkerPoolNamesUniqueNotDup(t *testing.T) {
 	}
 }
 
+func TestCheckIPXEScriptURL(t *testing.T) {
```

Comment:
Can we make table driven tests here to test all scenarios?
- OSVersion is NOT empty and IPXEScriptURL IS empty for worker: should give error
- OSVersion is NOT empty and IPXEScriptURL IS empty for controller: should give error
- OSVersion IS empty and IPXEScriptURL IS empty for worker: should be OK
- OSVersion IS empty and IPXEScriptURL IS empty for controller: should be OK
- OSVersion is NOT empty and IPXEScriptURL IS NOT empty for worker: should be OK
- OSVersion is NOT empty and IPXEScriptURL IS NOT empty for controller: should be OK

Reply:
Sure